src/main/java/domain/Car.java

```diff
@@ -4,24 +4,26 @@
 import strategy.MovableStrategy;
 
 public class Car {
-    private String player;
+    private final String player;
     private int location;
-    private MovableStrategy movableStrategy;
-
-    public Car(String player, MovableStrategy movableStrategy) {
-        this.player = player;
-        this.location = 0;
-        this.movableStrategy = movableStrategy;
-    }
+    private final MovableStrategy movableStrategy;
 
     public Car(String player, int location, MovableStrategy movableStrategy) {
         this.player = player;
         this.location = location;
         this.movableStrategy = movableStrategy;
     }
 
-    public void move(int randomNumber) {
-        if (movableStrategy.move(randomNumber)) {
+    public static Car of(String player, MovableStrategy movableStrategy) {
+        return new Car(player, 0, movableStrategy);
+    }
+
+    public static Car locationOf(String player, int location, MovableStrategy movableStrategy) {
```

Comment:
개인적으로 `locationOf` 메소드 명이 뭔가 location 객체를 결정하는 듯한 느낌이 듭니다:)
```suggestion
    public static Car createWithLocation(String player, int location, MovableStrategy movableStrategy) {
```
같은 건 어떨까요? :) 참고만 해주세요!